src/renderer.ts

```diff
@@ -782,11 +1287,19 @@ export const renderers: {[key: string]: ICellRendererFactory} = {
   min: combineCellRenderer,
   mean: combineCellRenderer,
   script: combineCellRenderer,
+  heatmapcustom: new HeatmapCellRenderer(),
+  threshold: new ThresholdCellRenderer(),
+  sparkline: new SparklineCellRenderer(),
+  verticalbar: new VerticalBarCellRenderer(),
+  boxplot: new BoxplotCellRenderer(),
```

Comment:
one plot is from the array and another is from directly with five number of box data